cmd/broker/fanout/main.go

```diff
@@ -52,6 +52,10 @@ type envConfig struct {
 	// continuous sync failures (or no sync at all) to be stale.
 	MaxStaleDuration time.Duration `envconfig:"MAX_STALE_DURATION" default:"1m"`
 
+	// MaxOutstandingBytes is the maximum size of unprocessed messages (unacknowledged but not yet expired).
+	// Default is 800Mb (~763Mi)
+	MaxOutstandingBytes int `envconfig:"MAX_OUTSTANDING_BYTES" default:"799999525"`
```

Comment:
Why the odd number here?

Reply:
No strong reason, other than myself tying to account for inconsistencies between Mebibytes and Megabytes used across different settings, and converting these back and forth in my notes. The difference is due to rounding and I've now rounded it up more precisely (the difference is negligible and does not affect the results).